src/frontend/next.config.js

```diff
@@ -24,6 +24,7 @@ const {
   OTEL_SERVICE_NAME = 'frontend',
   PUBLIC_OTEL_EXPORTER_OTLP_TRACES_ENDPOINT = '',
   NEXT_PUBLIC_SENTRY_DSN_CLIENT = '',
+  NEXT_PUBLIC_SENTRY_ENVIRONMENT = '',
```

Comment:
`SENTRY_ENVIRONMENT` is only consumed by the backend SDKs atm, the browser SDK does not consume this since none of our bundler plugins inject this automatically. Therefore this has to be passed directly to `Sentry.init`

Reply:
I think I'm passing it here: https://github.com/getsentry/opentelemetry-demo/pull/49/files#diff-78c8f261bd17cc8c5c2a5cbd2d75ce13ff40687aa89938147442c265090abf73R9

Or should I do something extra?

Reply:
Ah yes this is great - should be fine here.